Source/CMakeLists.txt

```diff
@@ -124,6 +124,4 @@ set(CMAKE_C_FLAGS_MINSIZEREL ${CMAKE_CXX_FLAGS_MINSIZEREL})
 add_subdirectory(Core)
 add_subdirectory(Tests)
 add_subdirectory(Tools)
-if(SC_WITH_CSHARP)
-    add_subdirectory(Wrapper)
-endif()
+add_subdirectory(Wrapper)
```

Comment:
Why move the if(SC_WITH_CSHARP) to Wrapper's CMakeLists?

Reply:
Hi @gongminmin, I guess that ShaderConductorWrapper will be included in future build artifacts, so If someone wants to use ShaderConductor from C# or another language only have to download the compiled artifacts.